examples/stable-diffusion/train_text_to_image_sdxl.py

```diff
@@ -1149,7 +1155,7 @@ def compute_time_ids(original_size, crops_coords_top_left):
 
                 # Gather the losses across all processes for logging (if we use distributed training).
                 avg_loss = accelerator.gather(loss.repeat(args.train_batch_size)).mean()
-                train_loss += avg_loss / args.gradient_accumulation_steps
+                train_loss = avg_loss / args.gradient_accumulation_steps
```

Comment:
can you also check what's the performance to go back to 
train_loss += avg_loss / args.gradient_accumulation_steps